parachain/config.json

```diff
@@ -27,11 +27,12 @@
 		{
 			"bin": "target/release/artemis",
 			"id": "200",
-			"wsPort": 9966,
-			"port": 30666,
```

Comment:
Can we rather just customize the config for E2E tests in the setup script? This config has uses outside of the E2E tests.

Reply:
sure, coming in next commit